src/views/Space/Overview.vue

```diff
@@ -78,6 +78,7 @@ const grouped = computed(() => {
           <Stamp
             :id="space.id"
             :size="90"
+            type="space-sx"
```

Comment:
Is this supposed to work with this PR?
https://github.com/snapshot-labs/stamp/pull/51/files

I don't see how this type ends up being used.

https://github.com/snapshot-labs/stamp/blob/f049ba2bd614729ece6b363195477bd8e5ddbddc/src/api.ts#L55-L57

Reply:
Yeah took space-sx  by mistake from [here](https://github.com/snapshot-labs/stamp/pull/51/files#diff-d43475a59a1dcd7ce7ecfab266b52211539c3d0148385f07485dd3de61a34964R8)